vorbis.go

```diff
@@ -210,6 +210,13 @@ func (m *metadataVorbis) Genre() string {
 }
 
 func (m *metadataVorbis) Year() int {
+	if len(m.c["year"]) != 0 {
```

Comment:
Would prefer to maintain existing behaviour when `date` is set (rather than overriding it when year and date are set).  Can this go after we have checked `date` instead?